src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -64,43 +76,43 @@ first "effective" echo and the center of the last "effective" echo.
-| NegativeContrast            | OPTIONAL. Boolean (`true` or `false`) value specifying whether increasing voxel intensity (within sample voxels) denotes a decreased value with respect to the contrast suffix. This is commonly the case when Cerebral Blood Volume is estimated via usage of a contrast agent in conjunction with a T2\* weighted acquisition protocol. |
+| **Key name**                | **Requirement level** | **Data type** | **Description**                                                                                                                                                                                                                                                                                                       |
+| --------------------------- | --------------------- | ------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| FlipAngle                   | RECOMMENDED           | [number][]    | Flip angle for the acquisition, specified in degrees. Corresponds to: DICOM Tag 0018, 1314 `Flip Angle`.                                                                                                                                                                                                              |
+| MultibandAccelerationFactor | RECOMMENDED           |               | The multiband factor, for multiband acquisitions.                                                                                                                                                                                                                                                                     |
+| NegativeContrast            | OPTIONAL              | [boolean][]   | `true` or `false` value specifying whether increasing voxel intensity (within sample voxels) denotes a decreased value with respect to the contrast suffix. This is commonly the case when Cerebral Blood Volume is estimated via usage of a contrast agent in conjunction with a T2\* weighted acquisition protocol. |
 
 #### Slice Acceleration
 
-| Field name                  | Definition                                                     |
-| ---------------------------- | ------------------------------------------------------------- |
-| MultibandAccelerationFactor | RECOMMENDED. The multiband factor, for multiband acquisitions. |
+| **Key name**                | **Requirement level** | **Data type** | **Description**                                   |
+| --------------------------- | --------------------- | ------------- | ------------------------------------------------- |
+| MultibandAccelerationFactor | RECOMMENDED           |               | The multiband factor, for multiband acquisitions. |
```

Comment:
Duplicate of previous table. Probably want to remove it from the previous and keep it here, going by the headers.